interface/super/edit_layout.php

```diff
@@ -1192,7 +1192,7 @@ function cidChanged(lino, seq) {
 // This invokes the popup to edit layout properties or add a new layout.
 function edit_layout_props(groupid) {
  dlgopen('edit_layout_props.php?layout_id=<?php echo attr($layout_id); ?>&group_id=' + groupid,
-  '_blank', 700, 550);
+  '_blank', 700, 550, '', 'Layout Properties');
```

Comment:
Looks like this title should be translated.

Reply:
Now Rod, how would it look if I didn't submit code that didn't need a translation or escape! :)